src/Bladeburner/Skill.ts

```diff
@@ -108,35 +102,31 @@ export class Skill {
      *
      * Define:
      *
-     * $$ e = -2 \ast b - 2 \ast c \ast d + c $$
+     * $$ m = -b - c \ast d + \frac{c}{2} $$
      *
-     * $$ Delta = \sqrt{a \ast (a \ast {e ^ 2} + 8 \ast c \ast y)} $$
+     * $$ Delta = \sqrt{{m ^ 2} + \frac{2 \ast c \ast y}{a}} $$
      *
      * Solutions:
      *
-     * $$ x_1 = \frac{a \ast e + Delta}{2 \ast a \ast c} $$
+     * $$ x_1 = \frac{m + Delta}{c} $$
      *
-     * $$ x_2 = \frac{a \ast e - Delta}{2 \ast a \ast c} $$
+     * $$ x_2 = \frac{m - Delta}{c} $$
      *
      * $a$, $c$ and $y$ are always greater than 0, so $x_2$ is always less than 0. Therefore, $x_1$ is the only
      * solution.
```

Comment:
I've been letting this sit for too long. Here's my attempt at a comment (it's far short of a true proof):

Due to floating-point rounding and edge-cases, we cannot ensure that rounding x_1 will give us the correct integer. In other words, we cannot be sure that x_1 is within 0.5 of the integer value we want. But, we *can* be sure that it is within *1* of the value we want, which means that checking the numbers above and below the rounded value are sufficient to find our correct integer.